src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,130 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String NAME = "name";
+    private static final String COUNTRY = "country";
+    private static final String ID = "id";
+    private static final int PARAMETER_FIRST_INDEX = 1;
+    private static final int PARAMETER_SECOND_INDEX = 2;
+    private static final int PARAMETER_THIRD_INDEX = 3;
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturer = new ArrayList<>();
+        String insertRequest = "SELECT * FROM manufacturers WHERE is_deleted = FALSE;";
```

Comment:
why request? It is query)) Same everywhere

Reply:
@Andriy-dz Bohdan wrote this on the video) I'll change it now)